routes/api/contacts.js

```diff
@@ -22,6 +26,13 @@ router.get("/", async (req, res, next) => {
 });
 
 router.get("/:id", async (req, res, next) => {
+  const { id } = req.params;
+  if (id.length !== 24) {
```

Comment:
Czy musimy tutaj mieć taki warunek?  Wydaje mi się, że jest niepotrzebny. W innych funkcjach też występuje.

Reply:
Ten warunek jest w sumie informacyjnie. Bez tego warunku jak wrzuci sie mniej znakow niz 24 to wywali nam po prostu {message: "not found"}